nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/cef/ValidateLocale.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.cef;
+
+import org.apache.nifi.components.ValidationContext;
+import org.apache.nifi.components.ValidationResult;
+import org.apache.nifi.components.Validator;
+
+import java.util.Locale;
+
+/**
+ * This class is identical to {@code org.apache.nifi.processors.standard.ParseCEF.ValidateLocale}.
```

Comment:
In light of this comment, did you consider creating a new module under `nifi-commons` to avoid duplicating this Validator class?

Reply:
I share your concerns with this and I was thinking heavily how to approach. Finally I decided this way considering the amount of duplicated code. The remark is more like something to keep in mind for now but I think in time they may evolve in different ways

Reply:
Thanks for reply, that seems reasonable given the narrow scope of this particular class.